erpnext/support/doctype/issue/issue.py

```diff
@@ -97,6 +98,123 @@ def split_issue(self, subject, communication_id):
 			doc.save(ignore_permissions=True)
 		return replicated_issue.name
 
+	def before_insert(self):
+		week = ['Monday', 'Tuesday', 'Wednesday', 'Thursday','Friday', 'Saturday', 'Sunday']
```

Comment:
You can make a common function to get weekdays... since it's used/created twice in this PR...